### DISCUSSION TITLE
CMake error occurred when compiling OBK on Kali Linux

### DISCUSSION THREAD

**Discussion by zawad1804** [OpenBangla/OpenBangla-Keyboard#303] [Q&A]
How to solve this problem? I am not used to with linux :) so i need you guys attention to run OBK on my system. Please help
https://user-images.githubusercontent.com/74665021/177837611-d85635a6-fe9f-4a57-b644-59e5fafced7b.png

**Comment by bdeshi** [comment]
You have missing dependencies. Read from here: https://github.com/OpenBangla/OpenBangla-Keyboard#compiling-on-gnulinux

or you can try to install prebuilt packages from here: https://github.com/OpenBangla/OpenBangla-Keyboard/releases/tag/2.0.0

**Reply by zawad1804** [reply]
I have all the updated dependencies installed in my system. can you please check the screenshot and tell me which dependencies i am missing?
# Do i need to install all the dependencies in the same directory where the OBK folder is?
I didn't find any prebuilt packages of OBK fro Kali Linux. can you tell me which prebuilt package will work in kali linux?

**Reply by bdeshi** [reply]
> I have all the updated dependencies installed in my system. can you please check the screenshot and tell me which dependencies i am missing?

from the error msg, it looks like you're missing at least qt5 base dev packages.

> Do i need to install all the dependencies in the same directory where the OBK folder is?

no

> I didn't find any prebuilt packages of OBK fro Kali Linux. can you tell me which prebuilt package will work in kali linux?

afaik, kalilinux is based on debian, so the .deb files should work.

**Comment by zawad1804** [comment]
Thanks vai i have installed OBK using .deb file. But the problem is OBK is not working in browsers. But it is working fine in other softwares as well as in linux search bar also. How can i write bangla in browser's with OBK??
